packages/theme/src/client/module/navbar/components/RepoLink.ts

```diff
@@ -35,7 +35,7 @@ export default defineComponent({
               // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
               h(resolveComponent(`${repo.value.type}Icon`), {
                 style: {
-                  width: "1.25rem",
+                  width: "2rem",
```

Comment:
What I want to express is that you may need to modify icon svg, specially when other icon is rendering well.